.github/workflows/ci.yml

```diff
@@ -37,8 +37,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        activerecord: [ '6.1', '7.0', '7.1' ]
-        ruby: [ '3.0', '3.1', '3.2' ]
+        activerecord: [ '6.1', '7.0', '7.1', '7.2' ]
+        ruby: [ '3.0', '3.1', '3.2', '3.3' ]
+        exclude:
+          - activerecord: '7.2'
+            ruby: '3.0'
```

Comment:
NOTE: activerecord 7.2 requires ruby 3.1 or greater. I added this exclusions so that we can continue to support ruby 3.0 for the time being.